docs/pages/database-access/rbac.mdx

```diff
@@ -1,10 +1,8 @@
 ---
-title: Database Access Access Controls
+title: Database Access Role-Based Controls
```

Comment:
This is the one change I'd like an independent review of.

Reply:
This looks a bit awkward to me. Would "Role-Based Access Controls for Databases" make sense here?

Reply:
How about 

Database Role-Based Access Controls ?

Reply:
```suggestion
title: Database Role-Based Access Controls
```